components/movementsensor/imuvectornav/imu.go

```diff
@@ -26,10 +26,10 @@ var model = resource.DefaultModelFamily.WithModel("imu-vectornav")
 
 // Config is used for converting a vectornav IMU MovementSensor config attributes.
 type Config struct {
-	SPI   string `json:"spi_bus"`
-	Speed *int   `json:"spi_baud_rate"`
-	Pfreq *int   `json:"polling_freq_hz"`
-	CSPin string `json:"chip_select_pin"`
+	SPI      string `json:"spi_bus"`
+	BaudRate *int   `json:"spi_baud_rate,omitempty"`
+	Pfreq    *int   `json:"polling_freq_hz,omitempty"`
+	CSPin    string `json:"chip_select_pin"` // this is the gpio number that we plugged it into
```

Comment:
The name for `CSPin` can be a little more descriptive, GPIOPin. We are not really selecting any chip here

Reply:
No we are, we're selecting a channel from the chip select, it cannot be any GPIO pin, renaming it would be wrong.

Reply:
I will revert the name

Reply:
Then the comment needs to reflect and explain what chip_select_pin means

Reply:
@susmitaSanyal comment has been updated